src/jllvm/materialization/ByteCodeCompileLayer.cpp

```diff
@@ -743,7 +775,7 @@ void CodeGen::codeGenBody(const Code& code)
             }
         }
 
-        codeGenInstruction(std::move(operation));
+        codeGenInstruction(operation);
```

Comment:
I know the variant is currently trivial so the move is unecessary, but that will probably be chanegd once the `switch` instructions are implemented, and it doesn't hurt to have it until then either.